app/components/header-links.js

```diff
@@ -59,10 +59,12 @@ export default Component.extend({
     return classes.join(' ');
   }),
 
+  enableAssemblaLogin: localStorage['enableAssemblaLogin'] == 'true',
```

Comment:
I assume this is only for testing and development? In this case, this should not go to production. Otherwise, probably it'd be better to use `config/environment` or feature flags for this purpose.